client/create-context-blocks.js

```diff
@@ -320,15 +320,27 @@ var searchIntegrations = {
     flickr: {
       methodName: 'flickrImageSearchList',
       mapFn: function(e) {
+        var username, uploadDate, title;
+        if (e.media) { 
+          //if single image result
+          ownername = e.owner.username;
+          uploadDate = e.dateuploaded;
+          title = e.title._content;
+        } else {
+          //if search result
+          ownername = e.ownername;
+          uploadDate = e.dateupload;
```

Comment:
that is so weird that it is `dateuploaded` vs `dateupload`. i assume that's a flickr api weirdness